common/transport/amqp/src/amqp_common_errors.ts

```diff
@@ -110,6 +110,30 @@ export function translateError(message: string, amqpError: Error): AmqpTransport
     // In the case of a invalid Twin object, this will return the generic error message plus the specific error messaged provided from the server.
     let errorString = message + '. ' + (<any>amqpError.message).message.toString();
     error = new Error(errorString);
+  } else if (amqpError.constructor?.name) {
+    //
+    // Questions for reviewers:
+    //
+    // 1. We don't want to translate these errors because we already have the right error,
+    //    but, what do we do with the passed-in message?  Do we lose that info, or do we
+    //    have some sort of "inner exception". e.g. if the passed error is
+    //    "NotConnectedError: SenderLink disconnected", and message is "Could not send",
+    //    What do we want to return?
```

Comment:
One possibility is to add a property to the error object called `innerError` with the passed in message. We'd have to leave a comment somewhere so the person debugging in the future knows that this message is there.